build.gradle

```diff
@@ -314,7 +323,25 @@ project(":ui") {
         }
     }
 
-
+    if(project.hasProperty('generation') || project.hasProperty('genonly')){
+        def cvLoc = System.getenv('OPEN_CV_LIB')
+        def jarLoc = System.getenv('OPEN_CV_JAR')
```

Comment:
Use a better variable name, `jarLoc` isn't very descriptive. `cvJarLocation` would be better. Ditto for `cvLoc` -- something like `cvNativeLibLocation` would be a lot better.